scifin/classifier/classifier.py

```diff
@@ -147,48 +153,57 @@ def kmeans_base_clustering(corr, names_features=None, max_num_clusters=10, n_ini
     -----
       Function adapted from "Machine Learning for Asset Managers",
       Marcos López de Prado (2020).
+
+      To learn more about sklearn.cluster.KMeans():
+      https://scikit-learn.org/stable/modules/generated/sklearn.cluster.KMeans.html#sklearn.cluster.KMeans
     """
     
     # Checks
     if not isinstance(max_num_clusters, int):
         raise AssertionError("max_num_clusters must be integer.")
```

Comment:
this is redundant if you use the @typecheck decorator

Reply:
True, but the AssertionError is more direct. @typeckeck leaves long error tracks before giving the message, so I see it more as at "catch-all errors" method than a full replacement... at least for the moment.

Reply:
Anyway it should be TypeError instead of AssertError as well

Reply:
True! I've corrected some others also.